environments/base/default-config.libsonnet

```diff
@@ -3,7 +3,7 @@
 
   name: 'observatorium-xyz',
   namespace: 'observatorium',
-  thanosVersion: 'master-2020-05-24-079ad427',  // Fixes a blocker issue in v0.13.0-rc.0
+  thanosVersion: 'master-2020-08-12-70f89d83',  // Fixes a blocker issue in v0.13.0-rc.0
```

Comment:
No need for this comment anymore, right? (:

Reply:
🙈 We should clean it